s3/replication/common/src/s3replicationcommon/aws_v4_signer.py

```diff
@@ -195,7 +195,8 @@ def prepare_signed_header(
             http_request,
             request_uri,
             query_params,
-            body):
+            body,
+            obj_range=None):
```

Comment:
for header signature, adding range is given

Reply:
%s/obj_range/read_range/gc